utils/resource/usage.go

```diff
@@ -84,6 +88,11 @@ func NewManager(diskPath string, frequency, cpuHalflife, diskHalflife time.Durat
 		onClose:            make(chan struct{}),
 		availableDiskBytes: math.MaxUint64,
 	}
+
+	metricsRegisterer := prometheus.NewRegistry()
+	processMetrics, _ := newMetrics("System resources", metricsRegisterer, m.processes)
```

Comment:
We should return the error here if one occurs.

Reply:
updated